composer.json

```diff
@@ -13,12 +13,13 @@
         "php": ">=5.5",
         "arbiter/arbiter": "^0.1",
         "destrukt/destrukt": "^0.7.1",
+        "equip/adr": "^1.0",
         "filp/whoops": "^1.1",
         "nikic/fast-route": "^0.7",
+        "monolog/monolog": "^1.17",
```

Comment:
Sure can, but will need to fix this file: https://github.com/equip/framework/pull/4/files#diff-4f99c1eaf57e677d457910a90234d70dR7